### DISCUSSION TITLE
Command to stop active aplay?

### DISCUSSION THREAD

**Discussion by stahlfaust** [gtxaspec/wz_mini_hacks#736] [Q&A] [answered]
How would one stop aplay? Does such a command exist?

To play: aplay <path/to/somefilename.wav> <volume 1-100>

To stop: aplay -s <path/to/somefilename.wav> ?

**Comment by virmaior** [comment, marked as answer]
```
kill $(ps | grep aplay | cut -d" " -f1 )
```

**Reply by stahlfaust** [reply]
Thank you!
